lib/logstash/pluginmanager/install.rb

```diff
@@ -56,6 +56,19 @@ def execute
     FileDependencies::Gem.hook
     options = {}
     options[:document] = []
+    if LogStash::Environment.test?
+      # This two options are the ones used to ask the rubygems to install
+      # all development dependencies as you can do from the command line
+      # tool.
+      #
+      # Comments from the command line tool.
+      # --development     - Install additional development dependencies
+      #
+      # Links: https://github.com/rubygems/rubygems/blob/master/lib/rubygems/install_update_options.rb#L150
+      #        http://guides.rubygems.org/command-reference/#gem-install
+      options[:dev_shallow] = true
+      options[:development] = true
+    end
```

Comment:
it already does like this is really clear, it make the code to install development dependencies. could you propose a comment thats clear to you?